src/Tribe/Commerce/Warnings.php

```diff
@@ -39,6 +40,24 @@ public function recurring_event_warning_msg( $post_id ) {
 		$this->show_notice( $warning );
 	}
 
+	/**
+	 * Add Provider missing warning for tickets.
```

Comment:
:no_entry_sign: Doc comment for parameter "$post_id" missing

<sub>hash: e07416325bcd02b11d397b544c332b12</sub>